Source/core/TokenizedStringList.h

```diff
@@ -0,0 +1,111 @@
+#pragma once
+#include "Module.h"
+
+namespace WPEFramework {
+namespace Core {
+
+    template <char SEPARATOR, bool REMOVE_WHITESPACES>
+    class TokenizedStringList {
```

Comment:
While looking at the code once more,  I saw a lot of "forwarding". I was wondering why don't you just inherit from std::vector<string> ? Than you can drop all forwarders and I do not think there is anything specific you would like to hide, right ?

Reply:
I tend not to use inheritance of std::vector, as the STL containers do not have virtual destructors, and that prevents polymorphic use of derived classes

Reply:
Good point !